tests/unit/htdocs/class/ZipfileTest.php

```
@@ -0,0 +1,361 @@
+        $minutes = ($result >> 5) & 0x3F;
+        // Seconds are stored as seconds/2
+        $seconds2 = $result & 0x1F;
+
+        $this->assertSame(14, $hours);
+        $this->assertSame(30, $minutes);
+        $this->assertSame(5, $seconds2, 'Seconds 10 stored as 10>>1 = 5');
+    }
+
+    public function testUnix2DosTimeReturnsDifferentValuesForDifferentDates(): void
+    {
+        $t1 = mktime(0, 0, 0, 1, 1, 2000);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $t1. Configured minimum length is 3. (undefined)

(ShortVariable)